<limits> not included in info.h

Hi, 
I just tried to compile the code, and run into this error:
Gpufit-master/Gpufit/info.cpp:122:82: error: ‘::max’ has not been declared; did you mean ‘std::max’?
  122 |     std::size_t const highest_size_t_value = std::numeric_limits< std::size_t >::max();
Googling a bit, I found that <limits> was not included in the header file. I think it would be good if you could do that, it seems that it is not included by default. Else, everything worked fine in the compilation - I will now run a few test runs.
Best regards,
    Stefan

--- a/Gpufit/info.cpp
+++ b/Gpufit/info.cpp
@@ -1,5 +1,6 @@
 #include "info.h"
 #include <algorithm>
+#include <limits>
 
 Info::Info() :
     n_parameters_(0),

